docs/articles_en/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training-tensorflow.rst

```diff
@@ -0,0 +1,132 @@
+Quantization-aware Training (QAT) with tensorFlow 2
```

Comment:
```suggestion
Quantization-aware Training (QAT) with TensorFlow
```

Reply:
let us change `TensorFlow 2` - > `TensorFlow`. That is because for reasons:
- TensorFlow 2 is not widely common term
- SavedModel format is the main format and other formats are not more like legacy

Reply:
CC: @alexsu52

Reply:
Done